interpreter/golang/readelf.go

```diff
@@ -94,7 +81,7 @@ func readString(x *pfelf.File, ptrSize int,
 //
 // It is guaranteed not to consume more than 1 MiB of memory.
 func ReadGoVersion(f *pfelf.File) (string, error) {
-	vs := getVersionSection(f)
+	vs := f.Section(".go.buildinfo")
```

Comment:
This still requires us to read the full section table and section strtab into memory. What are we actually gaining by doing this?

Reply:
Nevermind, I get now that the point isn't to avoid reading the section table, it's to avoid a call to `readBuildInfo` (which allocates 1MiB) below.